### PR TITLE
EclProblem: put drsdtcon with friends in separate class

### PR DESCRIPTION
**Pull request by akva2**
More separation of concerns.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — COMMENTED** [review]
I like the idea, but I think "Limits" is a poor name for this.  Something along the lines of "MixingControls" or "MixingRateLimits" makes more sense to me, given that this applies to the values of and the changes in Rs and/or Rv.

Any comments, @atgeirr and @totto82?

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
changed to EclMixingRateControls.

**Review by bska — COMMENTED** [review]
I noticed one very small stylistic issue.  Otherwise this looks good to me

File: ebos/eclmixingratecontrols.cc [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+    this->lastRs_ = rhs.lastRs_;
+    this->maxDRs_ = rhs.maxDRs_;
+    this->dRsDtOnlyFreeGas_ = rhs.dRsDtOnlyFreeGas_;
+
+    return *this;
+}
+
+template<class FluidSystem, class Scalar>
+void EclMixingRateControls<FluidSystem,Scalar>::
+init(size_t numDof, int episodeIdx, const unsigned ntpvt)
```
[the remark this answers is not in the document]
Comment (akva2) [reply ebos/eclmixingratecontrols.cc]: yeah, missed one.. fixed.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**